toqito/states/werner.py

```diff
@@ -2,10 +2,10 @@
 from __future__ import annotations
```

Comment:
Off comment, but do we need this `annotations` import?

Reply:
Not too sure about this because I have never used this before. 

From what I understand this is for those who might be migrating from one version of Python to another and are worried about backward compatibility. These imports make it easier to change the default behavior of the older version while also letting a maintainer use DeprecationWarning wherever needed. 

It seems ok to remove it for now. If there is some major change in Python, we can go back to using these imports.